content/docs/2.6/scalers/new-relic.md

```diff
@@ -0,0 +1,92 @@
++++
+title = "New Relic"
+availability = "2.6+"
+maintainer = "Community"
+description = "Scale applications based on New Relic NRQL"
+layout = "scaler"
+go_file = "newrelic_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `new-relic` trigger that scales based on a New Relic metric.
+
+```yaml
+triggers:
+  - type: new-relic
+    metadata:
+      # Required: Account - Subaccount to run the query on
+      account: 1234567
+      # Required: QueryKey - Api key to connect to New Relic
+      queryKey: "NRAK-xxxxxxxxxxxxxxxxxxxxxxxxxxx"
+      # Optional: nrRegion - Region to query data for
```

Comment:
Might be nice, to mention that `US` is default in here as well.

Reply:
shall I rename queryKey to userApiKey to make it more similar to the name it has on new relic?
In new relic this is 'ApiKey' of type 'User' it

Reply:
We now mention that US is the default in this yaml section